localization/english/bookmark/bookmark_583_l_english.yml

```diff
@@ -1,11 +1,26 @@
 
  #Llane Wrynn
  llane_wrynn_character:0 "King Llane"
- llane_wrynn_character_desc:0 "PH"
+ llane_wrynn_character_desc:0 "A knightly king, Llane was raised in the bond of brotherhood with his close friends knight-champion Anduin Lothar and Medivh, the Guardian. Alongside his friends and advisors, the young king defeated the trolls who killed his father during the Gurubashi War. Llane has proven himself true to the legacy of the Wrynn dynasty as Stormwind has become the mightiest it has ever been. With the invasion from Draenor incoming, Llane now has to face a threat more terrible than any of his ancestors did. The war to come may bring doom to Llane's entire people and legacy, but were he to succeed, he would be remembered as the greatest of all kings. Can his friends help him repel the incoming darkness?"
+
+ #Medivh
+ medivh_character:0 "Guardian Medivh"
+ medivh_character_desc:0 "Son of the court conjurer of Stormwind and of Aegwynn, the Guardian of Tirisfal, Medivh is the greatest scion of a spellcaster lineage. He became the hero of the kingdom during the Gurubashi War and became endeared with king Llane. Having succeeded his mother as Guardian, Medivh might be the most powerful mortal mage in the world. His ultimate task is to safeguard Azeroth from the direst threats. However, he has not been seen outside his lordship of Karazhan for some time. Many wonder what the Guardian has been up to.\n\nWill Medivh be able to save Stormwind once again? Does he have another plan? What fate awaits the tower of Karazhan in the middle of the war to come?"
+
+ #Blackhand
+ blackhand_character:0 "Warchief Blackhand"
+ blackhand_character_desc:0 "A force of nature, whose military prowess has earned him the nickname of 'destroyer', Blackhand is the chieftain of the Blackrock clan, the most powerful and populous clan of the Horde. Those two reasons led him to be chosen by his fellow chieftains and the influential warlock Gul'Dan to be warchief of the Horde and military overlord of the united orc clans. With Draenor conquered but the Horde starving of both combat and resources, the warchief is proud to lead his warriors into the conquest of Azeroth. The conquest will secure both new lands to rule and his position as the one true leader of the orcs despite the will of idealistic chieftains, ambitious commanders, and scheming warlocks...\n\nWill Blackhand bring Azeroth to its knees and achieve his path to glory as the uncontested ruler of the Horde? Will the humans or internal conflicts within the Horde put an end to the warchief's ambition? Whatever may happen, there will be bloodshed..."
```

Comment:
The text is too big, in-game it cuts out at "Whatever may happen"
![image](https://user-images.githubusercontent.com/74790101/223278236-6a121734-67f2-443b-8330-730f79495711.png)

Reply:
We could fix this with deleting one of the the "\n", so we don't have a free line but only a line break before "Will Blackhand..."

Reply:
```suggestion
 blackhand_character_desc:0 "A force of nature, whose military prowess has earned him the nickname of 'destroyer', Blackhand is the chieftain of the Blackrock clan, the most powerful and populous clan of the Horde. Those two reasons led him to be chosen by his fellow chieftains and the influential warlock Gul'Dan to be warchief of the Horde and military overlord of the united orc clans. With Draenor conquered but the Horde starving of both combat and resources, the warchief is proud to lead his warriors into the conquest of Azeroth. The conquest will secure both new lands to rule and his position as the one true leader of the orcs despite the will of idealistic chieftains, ambitious commanders, and scheming warlocks...\nWill Blackhand bring Azeroth to its knees and achieve his path to glory as the uncontested ruler of the Horde? Will the humans or internal conflicts within the Horde put an end to the warchief's ambition? Whatever may happen, there will be bloodshed..."
```